src/main/java/BuyStockUseCase/PortfolioGUI.java

```diff
@@ -28,12 +31,15 @@ public class PortfolioGUI extends JFrame implements iPortfolioGUI {
 
     public PortfolioGUI(Portfolio port, String username, boolean isComp) {
         super();
+        /**
+         * GUI where the user can view their portfolio information
+         */
         this.port = port;
         this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
```

Comment:
Not necessary but I would recommend adding a small comment here between these lines of code just to distinguish what's happening and make it less confusing. Refer to my branch for more clarification, but what I did was split up setting up jlabels and other code with a comment. Please let me know if you would like me to clarify better